src/org/exist/security/AbstractRealm.java

```diff
@@ -65,7 +63,7 @@
     protected final PrincipalDbByName<Group> groupsByName = new PrincipalDbByName<>();
     
 
-    private SecurityManager sm;
+    private SecurityManagerImpl sm;
```

Comment:
Why the Imp here and not the interface?

Reply:
That is one of the things I corrected from the original PR. Check the final state of the PR

Reply:
Yes sorry.....